recipes/PyFstat/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "PyFstat" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5275804036a0d814acf111857afd46fbb3cba81e48f3d5275fbdd7219d75dddc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >3.6,<3.9
+    - pip
+  run:
+    - python
```

Comment:
This line needs to match what is in host

Reply:
Ah, yes, that makes sense, thanks!

Reply:
The linter is happy now, thanks again.